examples/run.py

```diff
@@ -17,16 +17,19 @@
 
 
 def get_model_mp(data: Dict[str, pd.DataFrame],
-                 col_time: str = "time_start") -> Dict[str, ExcessMortalityModel]:
-    seas_spline_specs = SplineSpecs(knots=np.linspace(0.0, 1.0, 5),
-                                    degree=3,
-                                    r_linear=True,
-                                    knots_type="rel_domain")
-    time_spline_specs = SplineSpecs(knots=np.linspace(0.0, 1.0, 5),
-                                    degree=3,
-                                    knots_type="rel_domain")
+                 col_time: str = "time_start",
+                 units_per_year: int = 52,
+                 knots_per_year: float = 0.5) -> Dict[str, ExcessMortalityModel]:
     models = {}
     for name, df in data.items():
+        num_knots = max(2, int(knots_per_year*df.time.max()/units_per_year))
```

Comment:
It is possible to confirm that the placement of these knots falls on the same month or week every year? For example, would the knots fall on different months if the number of months in the dataset was not a multiple of 12?

Reply:
Hi Katie, it will be exactly the same, but I think it is ok, the goal is to roughly have one interior knots every other year, so that the interpolation and extrapolation will be stable for time trend.

Reply:
So you don't think having a knot one year in February and one year in August will be problematic? Given seasonality those months would be expected to be very different and my gut reaction is that that would place a bias on the time trend. But maybe I don't understand how splines work and there will be enough flexibility to work around this difference?

Reply:
The time trend is fitted based on the residual between the original deaths time series and seasonality pattern. Ideally it won't have seasonality effect in the residual anymore, so that the time trend is simply trend along the year axis :)

Reply:
If the spline is linear that makes me especially concerned about having knots in different months.

Reply:
That makes sense -- thanks for clarifying!